Apps/APN/src/data/sdk/CustomerIoSDKConfig.js

```diff
@@ -0,0 +1,52 @@
+import Env from '../../../env';
+
+export default class CustomerIoSDKConfig {
+  constructor({
+    siteId,
+    apiKey,
+    trackingUrl,
+    bqSecondsDelay,
+    bqMinNumberOfTasks,
+    trackScreens,
+    trackDeviceAttributes,
+    debugMode,
+  } = {}) {
+    this.siteId = siteId;
+    this.apiKey = apiKey;
+    this.trackingUrl = trackingUrl;
+    this.bqSecondsDelay = bqSecondsDelay;
+    this.bqMinNumberOfTasks = bqMinNumberOfTasks;
+    this.trackScreens = trackScreens;
+    this.trackDeviceAttributes = trackDeviceAttributes;
+    this.debugMode = debugMode;
+  }
+
+  static createDefault() {
+    return new CustomerIoSDKConfig({
+      siteId: Env.siteId,
+      apiKey: Env.apiKey,
+      trackingUrl: 'https://tracking.cio/',
```

Comment:
This URL doesn't point to the production API. Shouldn't the app's default SDK config point to production?

Reply:
It was a typo 🤐 [Fixed it](https://github.com/customerio/customerio-reactnative/pull/166/commits/1f90eb5723fc4cfd3020343da6836c61527895e5)